apps/minifront/src/components/dashboard/assets-table/index.tsx

```diff
@@ -11,20 +9,15 @@ import {
   TableRow,
 } from '@penumbra-zone/ui/components/ui/table';
 import { ValueViewComponent } from '@penumbra-zone/ui/components/ui/tx/view/value';
-import { abortLoader } from '../../../abort-loader';
 import { EquivalentValues } from './equivalent-values';
 import { Fragment } from 'react';
 import { shouldDisplay } from './helpers';
-
-export const AssetsLoader: LoaderFunction = async (): Promise<BalancesByAccount[]> => {
-  await abortLoader();
-  return await getBalancesByAccount();
-};
+import { useBalancesByAccount } from '../../../state/balances';
 
 export default function AssetsTable() {
-  const balancesByAccount = useLoaderData() as BalancesByAccount[];
+  const balancesByAccount = useBalancesByAccount();
```

Comment:
we were previously using `abortLoader` to sanity-check that certain preconditions are met before proceeding with a data-fetching operation. If any the checks failed, an exception was thrown, which aborted the loading process. What's the equivalent of this in `zquery` land?